text_scrubber/geo/find_in_string.py

```diff
@@ -95,19 +98,18 @@ def _find_in_string(sample: str, clean_func: Callable, normalize_func: Callable,
             # If we find any matches, we store the one with the highest score and additionally store the start and end
             # idx of the substring. Note that we add start_idx, to accommodate for the spaces after each word (which
             # were lost after calling .split())
+            threshold = match_threshold if len(combination) > threshold_small else match_threshold_small
```

Comment:
there is duplicate snippet of code 101-116  and 132-147... maybe you can create another function called `find_match`  or something in similar lines?

Reply:
Fair enough